frontend/app/templates/guide.hbs

```diff
@@ -25,4 +25,10 @@
             <img src="/images/guide_manual.png" class="guide_img" alt="Manual Data Image">
         </div>
     </div>
-</div>
\ No newline at end of file
+</div>
+
+
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.0/js/bootstrap.min.js"></script>
+<script src="{{rootURL}}js/jquery.minicolors.min.js"></script>
+<script src="{{rootURL}}js/script.js"></script>
+<script src="{{rootURL}}js/form.js"></script>
```

Comment:
We don't need this here.

Reply:
I have removed form.js and jquery but script.js is required otherwise it is not working